String_Object/Strings.py

```diff
@@ -0,0 +1,40 @@
+text = '''  tHis iz your homeWork, copy these Text to variable.
+
+ 
+
+  You NEED TO normalize it fROM letter CASEs point oF View. also, create one MORE senTENCE witH LAST WoRDS of each existING SENtence and add it to the END OF this Paragraph.
+
+ 
+
+  it iZ misspeLLing here. fix“iZ” with correct “is”, but ONLY when it Iz a mistAKE.
+
+ 
+
+  last iz TO calculate nuMber OF Whitespace characteRS in this Tex. caREFULL, not only Spaces, but ALL whitespaces. I got 87.'''
+cap_tex = ''
+# split the text in sentences and delete \n
+ls_text = text.split()
+new_text = ' '.join(ls_text)
```

Comment:
No need do create a new line for each transforamtion. You can do it inplace: ' '.join(text.split()) etc.